### PR TITLE
vc-authn-oidc 2.0 Deployment Configurations

### PR DESCRIPTION
**Pull request by i5okie**
- Add helm values for vc-authn-oidc helm chart deployments
- Update accredited-lawyer proof configurations

### CODE REVIEW

File: helm-values/vc-authn-oidc/dev.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    webhook-url: 'https://{{ include "vc-authn-oidc.host" . }}/webhooks'
+
+  ledgers.yml:
+    - id: SOVRINSandbox
+      is_production: true
+      is_write: true
+      genesis_url: 'https://raw.githubusercontent.com/sovrin-foundation/sovrin/stable/sovrin/pool_transactions_sandbox_genesis'
+    - id: BCovrinTest
+      is_production: true
+      genesis_url: 'http://test.bcovrin.vonx.io/genesis'
```
Comment (esune) [inline helm-values/vc-authn-oidc/dev.yaml:84]: Should this be BCovrin Dev? @WadeBarnes this matches the current configurations [here](https://github.com/bcgov/trust-over-ip-configurations/tree/main/openshift/templates/agent/config) but am wondering whether this is something we need to tweak.

Comment (WadeBarnes) [reply helm-values/vc-authn-oidc/dev.yaml:84]: It was set as BCovrin Test since most wallets don't ready from BCovrin Dev.

Comment (esune) [reply helm-values/vc-authn-oidc/dev.yaml:84]: Ok, I am good keeping it as-is then.

File: helm-values/vc-authn-oidc/dev.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
+ingressSuffix: -dev.apps.silver.devops.gov.bc.ca
+
+image:
+  repository: ghcr.io/bcgov/vc-authn-oidc
+  pullPolicy: IfNotPresent
+  tag: "2.0.0-beta1"
```
Comment (esune) [inline helm-values/vc-authn-oidc/dev.yaml:6]: Either remove the tag, or point at the latest available release, please.

File: helm-values/vc-authn-oidc/dev.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+ingress:
+  enabled: true
+  annotations:
+   route.openshift.io/termination: edge
+
+acapy:
+  image:
+    repository: ghcr.io/hyperledger/aries-cloudagent-python
+    pullPolicy: IfNotPresent
+    tag: py3.9-0.9.0
```
Comment (esune) [inline helm-values/vc-authn-oidc/dev.yaml:47]: Can you please upgrade to the latest `0.1.2` release? Or go with whatever is specified in the main chart.

File: helm-values/vc-authn-oidc/prod.yaml [hunk 4]
```diff
@@ -0,0 +1,94 @@
+ingressSuffix: .apps.silver.devops.gov.bc.ca
+
+image:
+  repository: ghcr.io/bcgov/vc-authn-oidc
+  pullPolicy: IfNotPresent
+  tag: "2.0.0-beta1"
```
Comment (esune) [inline helm-values/vc-authn-oidc/prod.yaml:6]: Same as previous comments.

File: helm-values/vc-authn-oidc/prod.yaml [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+ingress:
+  enabled: true
+  annotations:
+   route.openshift.io/termination: edge
+
+acapy:
+  image:
+    repository: ghcr.io/hyperledger/aries-cloudagent-python
+    pullPolicy: IfNotPresent
+    tag: py3.9-0.9.0
```
Comment (esune) [inline helm-values/vc-authn-oidc/prod.yaml:47]: Same as previous comments.

File: helm-values/vc-authn-oidc/test.yaml [hunk 6]
```diff
@@ -0,0 +1,94 @@
+ingressSuffix: -test.apps.silver.devops.gov.bc.ca
+
+image:
+  repository: ghcr.io/bcgov/vc-authn-oidc
+  pullPolicy: IfNotPresent
+  tag: "2.0.0-beta1"
```
Comment (esune) [inline helm-values/vc-authn-oidc/test.yaml:6]: Same as previous comments.

File: helm-values/vc-authn-oidc/test.yaml [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+ingress:
+  enabled: true
+  annotations:
+   route.openshift.io/termination: edge
+
+acapy:
+  image:
+    repository: ghcr.io/hyperledger/aries-cloudagent-python
+    pullPolicy: IfNotPresent
+    tag: py3.9-0.9.0
```
Comment (esune) [inline helm-values/vc-authn-oidc/test.yaml:47]: Same as previous comments.

**Comment by esune** [comment]
How did traction configuration changes end up in this PR?

**Review by esune — APPROVED** [review]

**Review by esune — APPROVED** [review]

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**